bridges/otelzap/core.go

```diff
@@ -200,6 +200,13 @@ func (o *Core) Write(ent zapcore.Entry, fields []zapcore.Field) error {
 	r.SetSeverityText(ent.Level.String())
 
 	r.AddAttributes(o.attr...)
+	if ent.Caller.Defined {
+		r.AddAttributes(
+			log.String("file", ent.Caller.File),
+			log.Int("line", ent.Caller.Line),
+			log.String("function", ent.Caller.Function),
```

Comment:
Can you please follow the same semantic conventions as in https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6253?

Reply:
Yeah sure. Do you mean that the attributes should not be added automatically or is it about the their attribute keys to use `semconv`?

Reply:
It  is it about the their attribute keys to use `semconv`. Sorry for the  lack of precision in my comment.

Reply:
It's fine no worries 😅 I am just not sure about using `semconv.CodeFilepathKey` because `ent.Caller.File` does not contain the file path but rather just the file name. Another fitting key from the attribute group does not exists sadly.

Reply:
Still it is a part of the filepath. I am good with it.